app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/summary/SummaryProductFragment.kt

```diff
@@ -282,7 +282,16 @@ class SummaryProductFragment : BaseFragment(), ISummaryProductPresenter.View {
                 Toast.makeText(requireContext(), err.message, Toast.LENGTH_SHORT).show()
             }
             binding.uploadingImageProgress.visibility = View.GONE
-            binding.uploadingImageProgressText.setText(R.string.image_uploaded_successfully)
+            when {
+                // toast to "add front image".
+                !sendOther -> {
+                    binding.uploadingImageProgressText.text = getString(R.string.front_image_uploaded_successfully)
+                }
+                // toast to "add additional image of packaging".
+                sendOther -> {
+                    binding.uploadingImageProgressText.text = getString(R.string.additional_image_uploaded_successfully)
+                }
+            }
```

Comment:
There is no need to use a `when` if there are only two cases, especially if they are one the negation of the other one. Please replace it with an `if-else`.
```suggestion
            binding.uploadingImageProgressText.text = if (sendOther) {
                     getString(R.string.additional_image_uploaded_successfully)
            } else {
                     getString(R.string.front_image_uploaded_successfully) 
            }
            
```